conftest.py

```diff
@@ -133,6 +133,17 @@ def get_coiled_software_name():
 
 @pytest.fixture(scope="session")
 def benchmark_db_engine(pytestconfig, tmp_path_factory):
+    """Session-scoped fixture for the SQLAlchemy engine for the benchmark sqlite database.
+
+    You can control the database name with the environment variable ``DB_NAME``,
+    which defaults to ``benchmark.db``. Most tests shouldn't need to include this
```

Comment:
When we say "most tests shouldn't need..." I'm assuming these are the tests using the `small_client` fixture right? If so, should say that? I think the way this sentence is, it is not clear when you need it or when you do not.

Reply:
I think it's any test that uses one of the "user-facing" benchmarking fixtures, most notably `benchmark_all` as the catch-all solution. See https://github.com/coiled/coiled-runtime/blob/main/tests/stability/test_spill.py#L11 for example. I'd probably go as far as making this a private fixture and saying that it's for internal use and should not be used in tests.